File: week_06/day_3/composition/Composition_Start/app/src/main/java/example/codeclan/com/composition_start/wizard_management/CleaningImplement.java
package example.codeclan.com.composition_start.wizard_management;

/**
 * Created by user on 28/08/2017.
 */

public abstract class CleaningImplement{

    String brand;

    public CleaningImplement(String brand) {
        this.brand = brand;
    }

    public String getBrand(){
        return this.brand;
    }
}

File: week_10/c_sharp_course/week_1/day_4/composition/checkpoint_1_composition/wizard_management/CleaningImplement.cs
namespace WizardManagement {
  public abstract class CleaningImplement {
    
    string brand;

    public string Brand
    {
        get { return brand; }
        set { brand = value; }
    }
  
    public CleaningImplement() {

    }
    
    public CleaningImplement(string brand) {
      this.brand = brand;
    }
    
  }
}